src/styles/main.scss

```
@@ -1,3 +1,18 @@
-body {
+* {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
```

you can add only box-sizing to `*` selector
```suggestion
* {
  box-sizing: border-box;
}
```